src/TrackingOptOut.php

```diff
@@ -116,7 +116,7 @@ public static function enqueueScriptsFirstHead() {
    * @implements wp_enqueue_scripts.
    */
   public static function wp_enqueue_scripts() {
-    wp_enqueue_script(Plugin::PREFIX . '-enable-opt_out', Plugin::getBaseUrl() . '/dist/scripts/enable-opt-out.js', ['jquery'], FALSE, TRUE);
+    wp_enqueue_script(Plugin::PREFIX . '-enable-opt_out', Plugin::getBaseUrl() . '/dist/scripts/enable-opt-out.js', ['jquery'], TRUE);
```

Comment:
Unless I'm mistaken, the intention here is to load the opt-out script after the content has been loaded, so that it can be applied to the ga-optout shortcode in the content body.

Did you check whether that still works correctly when loading the script in the header?

`TRUE` is not a supported value for the `$ver` argument of https://developer.wordpress.org/reference/functions/wp_enqueue_script/

I would think that the current value `FALSE` was intended:
> If version is set to false, a version number is automatically added equal to current installed WordPress version. 

```suggestion
    wp_enqueue_script(Plugin::PREFIX . '-enable-opt_out', Plugin::getBaseUrl() . '/dist/scripts/enable-opt-out.js', ['jquery'], FALSE, TRUE);
```